app/javascript/gobierto_data/webapp/components/Dataset.vue

```diff
@@ -446,6 +529,60 @@ export default {
       // TODO: indicar algo con el status OK
       console.log("postVisualization", status);
     },
+    getColumnsQuery(csv = '') {
+      const [ columns = '' ] = csv.split("\n");
+      this.arrayColumnsQuery = columns.split(",");
+    },
+    resetQuery(value) {
+      this.resetQueryDefault = value
+      if (value === true) {
+        this.isSavingPromptVisible = false
+        this.currentQuery = `SELECT * FROM ${this.tableName} LIMIT 50`;
```

Comment:
The default query is duplicated in 3 different places. Can we move it to a single place? (method?)

Reply:
I remove one of them. I can't remove `currenTquery` in `resetQuery` method because `currentQuery` is continuously updated and listen to any changes in the editor, so when reset the editor to initial state we need resetQuery to `SELECT * FROM ${this.tableName} LIMIT 50`

Reply:
Explore the idea about setting a `computed` prop, something like, `this.queryDatasetDefault`

Reply:
I tried, but it didn't work